apps/script/aichatCloudWatchDashboard/constants.js

```diff
@@ -0,0 +1,217 @@
+const DASHBOARD_NAME = 'GenAI';
```

Comment:
Can we make this "GenAICurriculum"? Just for clarity since we have a lot of AI projects across engineering right now. That would match the cloudformation stack I'm working on too